torch/onnx/_internal/onnxruntime.py

```diff
@@ -0,0 +1,881 @@
+import dataclasses
+import importlib
+import logging
+
+from functorch.compile import min_cut_rematerialization_partition
+
+import torch
+import torch._C
+import torch._ops
+import torch._prims.executor
+import torch.fx
+from torch._dynamo.backends.common import aot_autograd
+from torch._subclasses.fake_tensor import FakeTensor
+from torch.fx.passes.fake_tensor_prop import FakeTensorProp
+from torch.fx.passes.infra.partitioner import CapabilityBasedPartitioner
+from torch.fx.passes.operator_support import OperatorSupport
+from torch.fx.passes.tools_common import CALLABLE_NODE_OPS
+from torch.utils import _pytree
+
+try:
+    # Use try-except to initialize package-dependent global variables.
+    from typing import Any, Dict, List, Mapping, Optional, Set, Tuple, Union
+
+    import onnx
+    import onnxruntime  # type: ignore[import]
+    from onnxruntime.capi import _pybind_state as ORTC  # type: ignore[import]
+
+    # This is not use directly in DORT but needed by underlying exporter,
+    # so we still need to check if it exists.
+    importlib.import_module("onnxscript")
```

Comment:
You could use [torch.onnx._internal.exporter._assert_dependencies](https://github.com/pytorch/pytorch/blob/main/torch/onnx/_internal/exporter.py#L916) to check that

Reply:
I feel better no to touch anything under `torch.onnx` in this block.